tests/test_memory_repr.py

```diff
@@ -0,0 +1,129 @@
+import numpy as np
+import pytest
+import torch
+import cupy as cp
+from savant.utils.memory_repr import opencv_gpu_mat_as_cupy, cupy_as_opencv_gpu_mat
+from savant.utils.memory_repr_pytorch import opencv_gpu_mat_as_pytorch, pytorch_tensor_as_opencv_gpu_mat
+import cv2
+
+TORCH_TYPE = [torch.int8, torch.uint8, torch.float32]
```

Comment:
I think uint8 and float32 enough.

Reply:
I would suggest extending the checks for all supported types in the mapping dictionary.